### PR TITLE
FLP 301 GetCurrentPartyIds endpoint

### PR DESCRIPTION
**Pull request by tom-gough**

### CODE REVIEW

File: src/Domain/Repositories/ApprenticeshipQueryRepository.cs [hunk 1]
```diff
@@ -10,302 +10,326 @@
...
+        catch (Exception e)
+        {
+            _logger.LogError(e, "Error getting apprenticeships with episodes for provider UKPRN {Ukprn}", ukprn);
+        }
 
         return apprenticeshipWithEpisodes;
-     }
- }
\ No newline at end of file
+    }
```
Comment (tom-gough) [inline src/Domain/Repositories/ApprenticeshipQueryRepository.cs:310]: The indentation in this file was a mess, I fixed it all which is why it's flagged a mass of changes above. Below this line is the method I added for this ticket.

File: src/InnerApi/Controllers/ApprenticeshipController.cs [hunk 2]
```diff
@@ -1,13 +1,16 @@
+
```
Comment (MikeV-TalentConsulting) [inline src/InnerApi/Controllers/ApprenticeshipController.cs:1]: Whitespace

File: src/Domain/Repositories/ApprenticeshipQueryRepository.cs (hunk 1 above)
Comment (MikeV-TalentConsulting) [reply src/Domain/Repositories/ApprenticeshipQueryRepository.cs:310]: Fantastic, thank you

File: src/InnerApi/Controllers/ApprenticeshipController.cs (hunk 2 above)
Comment (tom-gough) [reply src/InnerApi/Controllers/ApprenticeshipController.cs]: sorted

**Review by MikeV-TalentConsulting — APPROVED** [review]

**Pull request merged**